Lib/WebHandler.py

```diff
@@ -139,23 +142,62 @@ def __processCredentials(self):
     """
     Extract the user credentials based on the certificate or what comes from the balancer
     """
+
+    if not self.request.protocol == "https":
+      return
+
+    # OIDC auth method
+    def oAuth2():
+      if self.get_secure_cookie("AccessToken"):
+        access_token = self.get_secure_cookie("AccessToken")
+        url = Conf.getCSValue("TypeAuths/%s/authority" % typeAuth) + '/userinfo'
```

Comment:
please check the url. because Conf.getCSValue("TypeAuths/%s/authority" % typeAuth) can be empty, if someone forget to add TypeAuths to CS.